--- a/Makefile
+++ b/Makefile
@@ -94,12 +94,9 @@
 	# install the desktop file
 	-mkdir -p $(DESTDIR)/usr/share/applications/
 	cp packaging/fedora/modrana.desktop $(DESTDIR)/usr/share/applications/
-	cp packaging/fedora/modrana-qt5.desktop $(DESTDIR)/usr/share/applications/
 	# install the startup scripts
 	-mkdir -p $(DESTDIR)/usr/bin
 	cp packaging/fedora/modrana $(DESTDIR)/usr/bin/
-	cp packaging/fedora/modrana-gtk $(DESTDIR)/usr/bin/
-	cp packaging/fedora/modrana-qt5 $(DESTDIR)/usr/bin/
 	# install the launcher
 	cp run/launcher/modrana $(DESTDIR)/usr/bin/
 
